tasks/db_deploy/test/manual_tests/sql/orca_schema_v4/remove.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS orca.providers, orca.collections, orca.provider_collection_xref, orca.granules, orca.files;
```

Comment:
You will should separate this out into individual DROP TABLE statements and put it in the correct order in order to properly cascade the removal of relationships.
- orca.files
- orca.granules
- orca.provider_collection_xref
- orca.collections
- orca.providers

In addition you should update the schema_versions table by removing the row for version 4 and updating the row for version 3 setting it to the most current version.

Reply:
makes sense.. removed